.github/workflows/generate-analysis.yml

```diff
@@ -0,0 +1,226 @@
+          pip install -r OCB-$TO_VERSION/requirements.txt
+          pip install ./OCB-$TO_VERSION
+          pip install -r server-tools-$TO_VERSION/requirements.txt
+          pip install -r openupgrade-$TO_VERSION/requirements.txt
+          # this is for l10n_eg_edi_eta which crashes without it
+          pip install asn1crypto
+          odoo -s -c odoo-$TO_VERSION.cfg --addons-path server-tools-$TO_VERSION,openupgrade-$TO_VERSION --stop-after-init
+      - name: Create previous Odoo database
+        env:
+          ODOO: env-${{ env.FROM_VERSION }}/bin/odoo -c odoo-${{ env.FROM_VERSION }}.cfg -d ${{ env.FROM_VERSION }}
+          ODOO_SHELL: env-${{ env.FROM_VERSION }}/bin/odoo shell -c odoo-${{ env.FROM_VERSION }}.cfg -d ${{ env.FROM_VERSION }}
+        run: |
+          $ODOO --without-demo=all -i upgrade_analysis --stop-after-init
+          $ODOO_SHELL <<EOF
+          install_wizard = env['upgrade.install.wizard'].create({})
+          install_wizard.select_odoo_modules()
+          install_wizard.install_modules()
+          env.cr.commit()
+          EOF
+          $ODOO_SHELL <<EOF
+          env['upgrade.generate.record.wizard'].create({}).generate()
+          env.cr.commit()
+          EOF
+      - name: Create current Odoo database
+        env:
+          ODOO: env-${{ env.TO_VERSION }}/bin/odoo -c odoo-${{ env.TO_VERSION }}.cfg -d ${{ env.TO_VERSION }}
+          ODOO_SHELL: env-${{ env.TO_VERSION }}/bin/odoo shell -c odoo-${{ env.TO_VERSION }}.cfg -d ${{ env.TO_VERSION }}
+        run: |
+          $ODOO --without-demo=all -i upgrade_analysis --stop-after-init
+          $ODOO_SHELL <<EOF
+          install_wizard = env['upgrade.install.wizard'].create({})
+          install_wizard.select_odoo_modules()
+          install_wizard.install_modules()
+          env.cr.commit()
+          EOF
+          $ODOO_SHELL <<EOF
+          env['upgrade.generate.record.wizard'].create({}).generate()
+          env.cr.commit()
+          EOF
+      - name: Delete all files to be generated from repository
+        env:
+          OPENUPGRADE_DIR: openupgrade-${{ env.TO_VERSION }}
+        run: |
+          for file in $OPENUPGRADE_DIR/openupgrade_scripts/scripts/*/*/{noupdate_changes.xml,upgrade_analysis.txt}; do
+            if [ ! -f $file ]; then
+              continue
+            fi
+            git -C $OPENUPGRADE_DIR rm ${file#$OPENUPGRADE_DIR/}
+          done
+      - name: Create analysis
+        env:
+          FROM_ODOO: env-${{ env.FROM_VERSION }}/bin/odoo -c odoo-${{ env.FROM_VERSION }}.cfg -d ${{ env.FROM_VERSION }}
+          TO_ODOO_SHELL: env-${{ env.TO_VERSION }}/bin/odoo shell -c odoo-${{ env.TO_VERSION }}.cfg -d ${{ env.TO_VERSION }}
+        run: |
+          $FROM_ODOO --max-cron-threads=0 &
+          ODOO_PID=$!
+          $TO_ODOO_SHELL <<EOF
+          config = env['upgrade.comparison.config'].create({
+              'database': '${{ env.FROM_VERSION }}',
```

Comment:
```suggestion
              'database': '${{ env.FROM_VERSION }}',
              'version':  '${{ env.FROM_VERSION }}',
```

Reply:
that's preset to the current version, isn't it?